### ISSUE TITLE
Cleanup JB GH Organization

### ISSUE THREAD

**Issue by elreydetoda** [JupiterBroadcasting/network#10]
tl;dr: remove [people](https://github.com/orgs/JupiterBroadcasting/people) who aren't with JB anymore from GH

---

### Details

So, I was going to try and see if I could make a new project for the [website repo](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com), but when I'd got to try and create one it said I'd needed to go to the organization level to create one.

![image](https://user-images.githubusercontent.com/10230166/194721050-fc7fd222-e676-4a18-b107-0b82281bb98d.png)

When there I figured I wouldn't be able to make one (because I'm not part of the JB organization), but that got me thinking... "I wonder who is"?

So, I went to check who's a part of the organization (at least [publicly](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership)) and saw a bunch of people who I haven't heard of... :thinking: 

![image](https://user-images.githubusercontent.com/10230166/194721001-f01fa767-9574-45d0-a9c3-ab04de08f39b.png)

By default all members have a specific set of capabilities (I created a temporary organization so I could see exactly what the defaults are), which are in the picture below:

![image](https://user-images.githubusercontent.com/10230166/194721379-730152b9-82a2-4600-991d-d74a3f273102.png)

While overall those aren't very harmful to @JupiterBroadcasting as a whole, if those have been modified for the organization it might give people (who might not be involved in JB anymore) dangerous permissions to JB's repos...

It does by default (which isn't show by the picture above) allow all members to [create new repos](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization#organization-members) (and [some other things](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization#permissions-for-organization-roles) (not too bad though with a quick glance IMO)) under the JB organization. So, while I don't know any of those people (and I'm sure they probably wouldn't do this) they might create something that could be profane or explicit which could give the JB organization some bad publicity at least. This would also allow them to be an administrator on the repo (because whoever creates a repo is the admin on it by default), so that might give them to some special permissions in the organization when running GH actions. (I can't think of a direct attack path for this currently but it doesn't seem like the best situation :upside_down_face: )

---

### Summary of to-do

So, all that to say I'd recommend cleaning up the people who aren't involved in the JB organization actively anymore. If you'd like to keep them on but restrict their privilege to create repos (which I'd recommend), then you can follow [this guide](https://docs.github.com/en/organizations/managing-organization-settings/restricting-repository-creation-in-your-organization). (I've never done that before, but it's in GH docs :upside_down_face: )

**Comment by gerbrent** [comment]
nice catch! We did clean up a little a couple months ago when the website work was just getting started, but clearly not enough. 

I'm on it!

**Comment by gerbrent** [comment]
....done!

Members list very much thinned, w a removeal of 22 historic and non-active members.

👍🏻
